packages/playwright-core/src/client/locator.ts

```diff
@@ -150,7 +154,7 @@ export class Locator implements api.Locator {
     return await this._frame._highlight(this._selector);
   }
 
-  locator(selectorOrLocator: string | Locator, options?: LocatorOptions): Locator {
+  locator(selectorOrLocator: string | Locator, options?: Omit<LocatorOptions, 'visible'>): Locator {
```

Comment:
Why don't we allow `visible` on the constructor?

Reply:
That's a new API I'm not sure we need. Being conservative here.